sdk/python/kfp/cli/pipeline.py

```diff
@@ -57,10 +63,16 @@ def upload(ctx, pipeline_name, package_file):
 )
 @click.argument("package-file")
 @click.pass_context
-def upload_version(ctx, package_file, pipeline_version, pipeline_id):
+def upload_version(ctx, package_file, pipeline_version, pipeline_id=None, pipeline_name=None):
     """Upload a version of the KFP pipeline"""
     client = ctx.obj["client"]
-
+    if pipeline_id == None and pipeline_name == None: #[TODO] do this using click instead... 
+        raise ValueError("Need to suppy 'pipeline-name' or 'pipeline-id'")
+    if pipeline_name!=None: 
```

Comment:
nit: shall we also raise a ValueError when both id and name are provided? (they may not be consistent, I think it's better we just avoid having both at the same time)